__tests__/app.test.js

```diff
@@ -35,6 +36,34 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an article object corresponding to the id in the endpoint", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
```

Comment:
This is a good start, but how do we know it is the actual article we are requesting? Instead of asserting that the article_id is any number, we should be asserting exactly what this is